### PR TITLE
chore(*): remove JBoss EAP 6 and AS 7 references

### PR DESCRIPTION
**Pull request by tmetzke**
related to CAM-14029

### CODE REVIEW

**Review by tasso94 — APPROVED** [review]
Looks good to me in general. 

Just one thought: The paths to the documents still contain `jboss` should we change this to `wildfly` as well? Example: `content/installation/full/jboss/_index.md`

**Comment by tmetzke** [comment]
Thought about it, but we are actually talking about WildFly and JBoss EAP here so technically it is still also JBoss and WildFly itself still uses a lot of thing that are named `jboss-xyz` so, I think it's not worth the effort.

**Comment by tasso94** [comment]
Shouldn't we then also keep the JBoss in the guides content-wise? Like this, JBoss EAP users might find things easier.

**Comment by tmetzke** [comment]
Yeah, good point. 👍 I'll adjust that a bit.

File: content/installation/full/jboss/_index.md [hunk 1]
```diff
@@ -1,21 +1,17 @@
 ---
 
-title: "Install the Full Distribution for JBoss/Wildfly"
+title: "Install the Full Distribution for Wildfly"
```
Comment (tmetzke) [inline content/installation/full/jboss/_index.md:3]:
```suggestion
title: "Install the Full Distribution for JBoss EAP/Wildfly"
```

File: content/installation/full/jboss/_index.md [hunk 2]
```diff
@@ -1,21 +1,17 @@
...
 weight: 20
 
 menu:
   main:
-    name: "JBoss/Wildfly"
+    name: "Wildfly"
```
Comment (tmetzke) [inline content/installation/full/jboss/_index.md:8]:
```suggestion
    name: "JBoss EAP/Wildfly"
```

File: content/installation/full/jboss/_index.md [hunk 3]
```diff
@@ -1,21 +1,17 @@
...
     identifier: "installation-guide-full-jboss"
     parent: "installation-guide-full"
-    pre: "Install the Full Distribution for JBoss or Wildfly Application Servers."
+    pre: "Install the Full Distribution for Wildfly Application Servers."
```
Comment (tmetzke) [inline content/installation/full/jboss/_index.md:11]:
```suggestion
    pre: "Install the Full Distribution for JBoss EAP or Wildfly Application Servers."
```

File: content/installation/full/jboss/configuration.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 
-title: 'Configure the Full Distribution for JBoss/Wildfly'
+title: 'Configure the Full Distribution for Wildfly'
```
Comment (tmetzke) [inline content/installation/full/jboss/configuration.md:3]:
```suggestion
title: 'Configure the Full Distribution for JBoss EAP/Wildfly'
```

File: content/installation/full/jboss/configuration.md [hunk 5]
```diff
@@ -13,17 +13,17 @@ menu:
 ---
 
 
-This page explains how to configure the full distribution for the JBoss/Wildfly application server.
+This page explains how to configure the full distribution for the Wildfly application server.
```
Comment (tmetzke) [inline content/installation/full/jboss/configuration.md:16]:
```suggestion
This page explains how to configure the full distribution for the JBoss EAP/Wildfly application server.
```

File: content/installation/full/jboss/configuration.md [hunk 6]
```diff
@@ -13,17 +13,17 @@ menu:
...
 
 
 ## LDAP
 
-In order to setup LDAP for the JBoss/Wildfly Application Server distribution, you have to perform the following steps:
+In order to setup LDAP for the Wildfly Application Server distribution, you have to perform the following steps:
```
Comment (tmetzke) [inline content/installation/full/jboss/configuration.md:21]:
```suggestion
In order to setup LDAP for the JBoss EAP/Wildfly Application Server distribution, you have to perform the following steps:
```

File: content/installation/full/jboss/configuration.md [hunk 7]
```diff
@@ -13,17 +13,17 @@ menu:
...
 
 
 ### Adjust the Process Engine Configuration
 
-Edit the file `standalone.xml` (or `domain.xml`) provided by the JBoss/Wildfly Application Server and add the [LDAP Identity Provider Plugin]({{< ref "/user-guide/process-engine/identity-service.md#the-ldap-identity-service" >}}) and the [Administrator Authorization Plugin]({{< ref "/user-guide/process-engine/authorization-service.md#the-administrator-authorization-plugin" >}}).
+Edit the file `standalone.xml` (or `domain.xml`) provided by the Wildfly Application Server and add the [LDAP Identity Provider Plugin]({{< ref "/user-guide/process-engine/identity-service.md#the-ldap-identity-service" >}}) and the [Administrator Authorization Plugin]({{< ref "/user-guide/process-engine/authorization-service.md#the-administrator-authorization-plugin" >}}).
```
Comment (tmetzke) [inline content/installation/full/jboss/configuration.md:26]:
```suggestion
Edit the file `standalone.xml` (or `domain.xml`) provided by the JBoss EAP/Wildfly Application Server and add the [LDAP Identity Provider Plugin]({{< ref "/user-guide/process-engine/identity-service.md#the-ldap-identity-service" >}}) and the [Administrator Authorization Plugin]({{< ref "/user-guide/process-engine/authorization-service.md#the-administrator-authorization-plugin" >}}).
```

File: content/installation/full/jboss/manual.md [hunk 8]
```diff
@@ -1,141 +1,26 @@
 ---
 
-title: 'Install the Full Distribution on a JBoss/Wildfly Application Server manually '
+title: 'Install the Full Distribution on a Wildfly Application Server manually '
```
Comment (tmetzke) [inline content/installation/full/jboss/manual.md:3]:
```suggestion
title: 'Install the Full Distribution on a JBoss EAP/Wildfly Application Server manually '
```

File: content/installation/full/jboss/manual.md [hunk 9]
```diff
@@ -1,141 +1,26 @@
...
 ---
 
 
-This document describes the installation of Camunda Platform and its components on a vanilla [JBoss EAP 6](https://developers.redhat.com/products/eap/download) or vanilla [Wildfly Application Server / JBoss EAP 7](http://www.wildfly.org).
+This document describes the installation of Camunda Platform and its components on a vanilla [Wildfly Application Server / JBoss EAP 7](http://www.wildfly.org).
 
 {{< note title="Reading this Guide" class="info" >}}
 This guide uses a number of variables to denote common path names and constants:
-`$JBOSS_HOME`/`$WILDFLY_HOME` points to the JBoss/Wildfly application server main directory.
+`$WILDFLY_HOME` points to the Wildfly application server main directory.
```
Comment (tmetzke) [inline content/installation/full/jboss/manual.md:20]:
```suggestion
`$WILDFLY_HOME` points to the JBoss EAP/Wildfly application server main directory.
```

**Review by tmetzke — COMMENTED** [review]
I would mainly adjust this in the installation sections, as we otherwise usually don't refer to JBoss EAP explicitly.

**Comment by tasso94** [comment]
Looks good!

**Pull request merged**